tests/testthat/test-Samples-methods.R

```diff
@@ -1,3 +1,5 @@
+# nolint start
```

Comment:
we should not need this here. this file should pass linter checks

Reply:
Not my option!  But have removed the directive and linted the file.  All should be good.